doc/MANUAL.md

```diff
@@ -117,20 +117,22 @@ which is useful if you're trying the latest code from
       --disable-validation-check
                             skips checking xml files against schema
       --offline             operate in offline mode
-      --detailed            display detailed report
+      --detailed            add CVE description in csv or json report (no effect on console, html or pdf)
 
     CVE Data Download:
       Arguments related to data sources and Cache Configuration
 
-      -n {api,api2,json-nvd,json-mirror}, --nvd {api,api2,json-nvd,json-mirror}
+      -n {api,api2,json,json-mirror,json-nvd}, --nvd {api,api2,json,json-mirror,json-nvd}
```

Comment:
@terriko I was wondering should we remove "api" its been time since we deprecated nvd api 1